app/src/androidTest/java/org/mozilla/focus/activity/SwitchLocaleTest.kt

```diff
@@ -1,37 +1,35 @@
-/* -*- Mode: Java; c-basic-offset: 4; tab-width: 20; indent-tabs-mode: nil; -*-
- * This Source Code Form is subject to the terms of the Mozilla Public
+/* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 package org.mozilla.focus.activity
 
 import android.os.Build.VERSION
-import androidx.test.espresso.action.ViewActions
-import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner
 import androidx.test.platform.app.InstrumentationRegistry
-import androidx.test.uiautomator.UiObjectNotFoundException
-import androidx.test.uiautomator.UiScrollable
-import androidx.test.uiautomator.UiSelector
 import org.junit.After
-import org.junit.Assert
 import org.junit.Ignore
 import org.junit.Rule
 import org.junit.Test
 import org.junit.rules.TestRule
 import org.junit.rules.TestWatcher
 import org.junit.runner.Description
-import org.junit.runner.RunWith
-import org.mozilla.focus.helpers.EspressoHelper.openMenu
-import org.mozilla.focus.helpers.EspressoHelper.openSettings
+import org.mozilla.focus.activity.robots.homeScreen
 import org.mozilla.focus.helpers.MainActivityFirstrunTestRule
-import org.mozilla.focus.helpers.TestHelper
-import org.mozilla.focus.helpers.TestHelper.pressBackKey
-import org.mozilla.focus.helpers.TestHelper.waitingTime
+import org.mozilla.focus.helpers.TestHelper.exitToTop
+import org.mozilla.focus.helpers.TestHelper.verifyTranslatedTextExists
 import java.util.Locale
 
-// This test checks all the headings in the Settings menu are there
-// https://testrail.stage.mozaws.net/index.php?/cases/view/47587
-@RunWith(AndroidJUnit4ClassRunner::class)
+// Testing some translated elements when switching locales and using system default
 class SwitchLocaleTest {
+    private val frenchLocaleinEn = "Français"
+    private val frenchLanguageMenu = "Langue"
+    private val frenchMotto = "Navigation privée automatique.\nNaviguez. Effacez. Recommencez."
+    private val frenchSettings = "Paramètres"
+    private val frenchHelp = "Aide"
+    private val frenchGeneralHeading = "Général"
+    private val frenchSystemLocale = "Valeur par défaut du système"
+    private val englishLanguageMenu = "Language"
+    private val englishLocaleInFr = "English (United States)"
+
```

Comment:
Is there any other way to access these values? I'm a tad hesitant to be defining these in a test, but don't let this block from seeing these green for now.

Reply:
Moved these to a StringsHelper, hope that's ok.